From 06d48081810aa67b250f81c23be729237848be25 Mon Sep 17 00:00:00 2001
From: Eric Naim <dnaim@cachyos.org>
Date: Thu, 12 Feb 2026 12:47:37 +0800
Subject: [PATCH 7/7] r8169: Also add CONFIG_R8125_MODULE

when r8125 is built as a module, whats defined is actually
CONFIG_R8125_MODULE instead of CONFIG_R8125.

Signed-off-by: Eric Naim <dnaim@cachyos.org>
---
 drivers/net/ethernet/realtek/r8169_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/realtek/r8169_main.c b/drivers/net/ethernet/realtek/r8169_main.c
index edb178711..32648463d 100644
--- a/drivers/net/ethernet/realtek/r8169_main.c
+++ b/drivers/net/ethernet/realtek/r8169_main.c
@@ -227,7 +227,7 @@ static const struct pci_device_id rtl8169_pci_tbl[] = {
 	{ 0x0001, 0x8168, PCI_ANY_ID, 0x2410 },
 	{ PCI_VDEVICE(REALTEK,	0x8126) },
 	{ PCI_VDEVICE(REALTEK,	0x8127) },
-#ifndef CONFIG_R8125
+#if !defined(CONFIG_R8125) && !defined(CONFIG_R8125_MODULE)
 	{ PCI_VDEVICE(REALTEK,	0x8125) },
 	{ PCI_VDEVICE(REALTEK,	0x8162) },
 	{ PCI_VDEVICE(REALTEK,	0x3000) },
-- 
2.53.0

